homeassistant/components/ssdp/__init__.py

```diff
@@ -618,6 +632,29 @@ async def async_get_discovery_info_by_udn(self, udn: str) -> list[SsdpServiceInf
             if ssdp_device.udn == udn
         ]
 
+    async def _handle_config_entry_removed(
```

Comment:
We can use the cache since it will already be populated if its in `self._ssdp_devices`

https://github.com/bdraco/home-assistant/commit/ee98007677beb49c265b01057fdc744bd02012e2

Reply:
Sounds good, please go ahead and push that change 👍